lib/list_course_manager.rb

```diff
@@ -39,6 +40,17 @@ def add_instructor_real_names
     end
   end
 
+  def add_classroom_program_manager_if_exists
+    cpm = SpecialUsers.classroom_program_manager
+    if cpm
+      CoursesUsers.create(user: cpm,
+                          course: @course,
+                          role: CoursesUsers::Roles::WIKI_ED_STAFF_ROLE,
+                          real_name: cpm.real_name,
+                          role_description: 'Wiki Ed Staff')
```

Comment:
I think role_description is better left out. We don't normally enter it for staff, and if present it will show up in the list of users in the Details section, and it duplicates the label "Wiki Ed Staff" that is already at the front of the list for this role.